src/shared/logger/activation.ts

```diff
@@ -125,6 +127,9 @@ export function makeLogger(
     if (opts.useDebugConsole) {
         logger.logToDebugConsole()
     }
+    if (opts.useConsoleLog && isInBrowser()) {
```

Comment:
is the isInBrowser check still needed? if something sets `useConsoleLog` why ignore it?

```suggestion
    if (opts.useConsoleLog) {
```

Reply:
We only used the console log for testing previously, but yeah this makes sense we should just enable it permanently for now.